bin/MadGraph5_aMCatNLO/submit_cmsconnect_gridpack_generation.sh

```diff
@@ -222,8 +227,9 @@ input_files="input_${card_name}.tar.gz"
 patches_directory="./patches"
 utilities_dir="./Utilities"
 plugin_directory="./PLUGIN"
+addons_dir="./addons"   # awightma
 if [ -e "$input_files" ]; then rm "$input_files"; fi
-tar -zchf "$input_files" "$card_dir" "$patches_directory" "$utilities_dir" "${plugin_directory}"
+tar -zchf "$input_files" "$card_dir" "$patches_directory" "$utilities_dir" "${plugin_directory}" "${addons_dir}"
```

Comment:
this is not safe since not all people are going to be using a model that lives under addons directory. why not just steer it with `extramodels.dat` ? 

e.g.
https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/SingleLepton_TypeIHeavyN/DYtoLNuJJ/DYtoLNuJJ_TypeIHeavyN-El_MN500_TuneCP5_13TeV-amcatnlo-pythia8/DYtoLNuJJ_TypeIHeavyN-El_MN500_TuneCP5_13TeV-amcatnlo-pythia8_extramodels.dat

this will be taken from [link](https://cms-project-generators.web.cern.ch/cms-project-generators/)

Reply:
Thanks for explaining, I've removed this. Our current model is not in https://cms-project-generators.web.cern.ch/cms-project-generators/. Can we add it or can only people in the gen group make changes?

Reply:
Should ask GEN conveners